main/resource-access.bs

```diff
@@ -106,6 +130,9 @@ Issue: Write Web Access Control section.
 Draft:
 A Solid data pod MUST conform to the Web Access Control specification [[!WAC]].
 
+A resource can advertise an ACL document that is directly associated by using the HTTP `Link` header with a `rel` value of `acl`. [[Source](https://github.com/solid/specification/issues/31#issuecomment-548360553)]
```

Comment:
Add TODO: registration.

Reply:
Do you mean IANA registration for "acl"? That and possibly the URI for "acl" is covered in the other issues eg. https://github.com/solid/specification/issues/54 . It will make its way in.